docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -683,11 +700,11 @@ This has been tested with LXC 3.0 and may or may not work with older versions.
    lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file
    ```
 
-3. Restart your container and install the required drivers in your container.
+3. Restart your container and [install the required drivers in your container](./index.md#configure--verify-hardware-acceleration).
 
-4. Add `jellyfin` user to the `video`, `render` and/or `input` groups depending on who owns the device inside the container.
+4. Add the `jellyfin` user to the group you chose in Step 2, i.e. the group that owns the DRI render device inside the LXC.
 
-5. Configure Jellyfin to use QSV or VA-API acceleration and change the default GPU `renderD128` if necessary.
+5. Configure Jellyfin to use QSV or VA-API acceleration and change the default GPU to `renderD128` if necessary.
```

Comment:
`renderD128` is the default, users are supposed to change it to something else if they have multiple GPUs and the one they wish to use isn't `renderD128`